tools/dm-task-configuration-file-intro.md

```diff
@@ -9,71 +9,77 @@ category: tools
 This document introduces the task configuration file of Data Migration --
 [`task.yaml`](https://github.com/pingcap/tidb-tools/blob/docs/docs/dm/zh_CN/configuration/task.yaml), including [Global configuration](#global-configuration) and [Instance configuration](#instance-configuration).
 
-For description of configuration items, see [Data Migration Task Configuration Options](../tools/dm-task-config-argument-description.md).
+For the feature and configuration of each configuration item, see [Data Synchronization Features](../tools/dm-data-synchronization-intro.md).
 
 ## Important concepts
 
-For description of important concepts including `instance-id` and the DM-worker ID, see [Important concepts](../tools/dm-configuration-file-overview.md#important-concepts). 
+For description of important concepts including `instance-id` and the DM-worker ID, see [Important concepts](../tools/dm-configuration-file-overview.md#important-concepts).
 
 ## Global configuration
 
-### Basic information configuration
+### Basic configuration
 
-```
+```toml
```

Comment:
the red background color is added by this `toml` language flag?

Reply:
@csuzhangxc It should be `yaml` here. Sorry about this red color. I've fixed it in a new commit.